runner/app/live/streamer/streamer.py

```diff
@@ -55,10 +55,10 @@ async def start(self, params: dict):
             run_in_background("ingress_loop", self.run_ingress_loop()),
             run_in_background("egress_loop", self.run_egress_loop()),
             run_in_background("report_status_loop", self.report_status_loop()),
-            run_in_background("control_loop", self.run_control_loop()),
         ]
         # auxiliary tasks that are not critical to the supervisor, but which we want to run
         self.auxiliary_tasks = [
+            run_in_background("control_loop", self.run_control_loop()),
```

Comment:
This change makes the control_url optional which is important for manual testing, do we need to move it back to `self.main_tasks`? It would be convenient to keep this change https://github.com/livepeer/ai-runner/blob/a1e1b33654870e06377b71862bc15bddf6e63b99/runner/app/live/streamer/streamer.py#L54-L59

Reply:
Yeah I even though I couldn't really understand the need for the change that made it an aux task [1], let's keep the `control_url` on `main_tasks` to avoid unexpected regressions. 

[1] Josh mentioned it made the shutdown more efficient, which i think could be because the control channel is closed before the others on stream close

Reply:
Moved back to main_tasks https://github.com/livepeer/ai-runner/commit/aa845528a4b04236de7705d4f916a88c0d61224a. 
Maybe we can have a "switch" to turn this on/off for testing later or mock the control_url using a testing script without requiring go-livepeer to generate a control stream